OpstalentCrudBundle.php

```diff
@@ -1,10 +1,16 @@
 <?php
+
+/**
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
```

Comment:
why not in class' docblock?

Reply:
https://pear.php.net/manual/en/standards.sample.php it could be both but for me @author could be separated (later we could change that to some package description  + author like on this link